src/queries/display-value.ts

```diff
@@ -6,30 +6,37 @@ import {
   fuzzyMatches,
   makeNormalizer,
   buildQueries,
+  Matcher,
+  MatcherOptions,
 } from './all-utils'
 
+function getElementValue(element: Element): string | undefined {
+  return (element as any).value
+}
+
 function queryAllByDisplayValue(
-  container,
-  value,
-  {exact = true, collapseWhitespace, trim, normalizer} = {},
+  container: HTMLElement,
+  value: Matcher,
+  {exact = true, collapseWhitespace, trim, normalizer}: MatcherOptions = {},
```

Comment:
I wonder if at some point we can deprecated exact=false since we now support regex

```diff
- queryAllByDisplayValue(node, 'xyz', { exact: false })
+ queryAllByDisplayValue(node, /xyz/)
```